### DISCUSSION TITLE
Entities kombinieren und gleichzeit anzeigen lassen

### DISCUSSION THREAD

**Discussion by DSRocker** [pixelit-project/PixelIt#282] [General]
Hallo zusammen,
können mehrere Werte (von einer Entität aus Home Assistant geliefert) verknüpft werden und gleichzeitig angezeigt werden?
=> Wert1: A / Wert2: B / Wert 3: C
=> Display zeigt dann: "A B C"
alle Werte sollten eigene Farbe bekommen.

vielen Dank!

**Comment by foorschtbar** [comment]
Das musst du irgendwie in HA lösen. Mit Node-RED ist das kein Problem :)

**Reply by DSRocker** [reply]
Wie geht das in Nodered?

**Reply by foorschtbar** [reply]
Oh, das mit der Farbe geht auch in Node-RED nicht, sry.

**Reply by DSRocker** [reply]
Das ist schade. Danke
